app/v4/amendForeignPropertyPeriodSummary/def1/Def1_AmendForeignPropertyPeriodSummaryRulesValidator.scala

```diff
@@ -142,7 +142,7 @@ object Def1_AmendForeignPropertyPeriodSummaryRulesValidator extends RulesValidat
           case None => valid
           case Some(_) =>
             expenses match {
-              case AmendForeignNonFhlPropertyExpenses(None, None, None, None, None, None, None, None, None, Some(_)) => valid
+              case AmendForeignNonFhlPropertyExpenses(None, None, None, None, None, None, _, _, None, Some(_)) => valid
```

Comment:
As this is a "behaviour" change, feels like a test ought to fail/be made to pass, to cover it. Same with the Def2?